### ISSUE TITLE
Autoplacing of the dropdown menu in the Select

### ISSUE THREAD

**Issue by MarikTar** [PeculiarVentures/peculiar-react-components#82]
When the dropdown menu fails to place it automatically.

![image](https://user-images.githubusercontent.com/47363177/98107949-e545ee00-1ea3-11eb-8994-bbbf3f71a174.png)

**Comment by donskov** [comment]
@MarikTar Try to use `lib-react-components@1.2.1-alpha.0`.
Changes:
1. Added Select dropdown auto-placement.
2. Added Autocomplete dropdown auto-placement.

Please test

**Comment by MarikTar** [comment]
@donskov, thanks, i tried it and can suggest the following:
1. placement : 'top' | 'bottom' | 'auto' = 'bottom' not all dropdown need apply automatically orientation for example:

| bottom | auto |
|---|---|
|![image](https://user-images.githubusercontent.com/47363177/98536412-7516de00-2290-11eb-96f4-ccaeff671f5b.png)|![image](https://user-images.githubusercontent.com/47363177/98536529-a42d4f80-2290-11eb-85fd-cdc6a0695fa8.png)|

or in mobile version.
it also prevents that if someone uses this library will face the problem that the "placement" is no longer available.

2. on mobile i have next problem
![image](https://user-images.githubusercontent.com/47363177/98537184-8b716980-2291-11eb-8e50-d27258fce06c.png)

3. if page have fixed "footer" happened next (can use z-index but this not good solution):

| no footer | have footer |
|---|---|
|![image](https://user-images.githubusercontent.com/47363177/98537562-194d5480-2292-11eb-9953-4c7f37395b14.png)|![image](https://user-images.githubusercontent.com/47363177/98537770-70ebc000-2292-11eb-9598-1fdc96cda752.png)|

P.S. I do not know if it will be appropriate, but I would add an auto-flip of the arrow. 
![image](https://user-images.githubusercontent.com/47363177/98538032-d770de00-2292-11eb-8978-c58d5358aa17.png)

**Comment by donskov** [comment]
@MarikTar Published `lib-react-components@1.2.1-alpha.2`

1. Fixed dropdown `left` position.
2. Added props:
```jsx
/**
 * Component dropdown start opened direction.
 */
placement: PropTypes.oneOf(['top', 'bottom']),
/**
 * If `true`, component will automatically calc possible dropdown opened direction.
 */
flip: PropTypes.bool,
```

Please try to use

**Comment by donskov** [comment]
Added auto-flip for the arrow
![image](https://user-images.githubusercontent.com/9417251/104334937-9ba8fd80-54fb-11eb-92ea-27c138659e3e.png)
![image](https://user-images.githubusercontent.com/9417251/104334958-a1064800-54fb-11eb-9da2-c7452d485b70.png)
